### PR TITLE
Bartnik wierzejska

### PR DESCRIPTION
**Pull request by NikodemBartnik**

### CODE REVIEW

**Review by swietjak — APPROVED** [review]
Cool, very nice

File: data/dataset.zip
Comment (SzymonSiakala) [inline data/dataset.zip:1]: You deleted the dataset, I can't merge it.

**Review by SzymonSiakala — CHANGES_REQUESTED** [review]

**Comment by NikodemBartnik** [comment]
pls approve

**Review by SzymonSiakala — APPROVED** [review]
Very cool, very nice

**Pull request merged**